service-front/composer.json

```diff
@@ -10,11 +10,12 @@
     ],
     "homepage": "https://github.com/ministryofjustice/opg-lpa",
     "require": {
+        "php": "^8.1",
```

Comment:
Still a problem here, as you now have two entries for PHP.

Reply:
Fixed now, anyway